test/src/userobjects/ElemSideNeighborLayersTester.C

```diff
@@ -63,13 +66,20 @@ GhostUserObject::execute()
 }
 
 void
-GhostUserObject::finalize()
+ElemSideNeighborLayersTester::finalize()
 {
   _communicator.set_union(_ghost_data);
+  _communicator.set_union(_evaluable_data);
 }
 
 unsigned long
-GhostUserObject::getElementalValue(dof_id_type element_id) const
+ElemSideNeighborLayersTester::getElementalValueLong(dof_id_type element_id,
+                                                    const std::string & field_name) const
 {
-  return _ghost_data.find(element_id) != _ghost_data.end();
+  if (field_name == "evaluable")
+    return _evaluable_data.find(element_id) != _evaluable_data.end();
+  else if (field_name == "ghosted")
+    return _ghost_data.find(element_id) != _ghost_data.end();
```

Comment:
Return `boolean` as an `unsigned long`?

Reply:
Now this is OK - Booleans fit into other interger types in a well-defined way. We are just going to use them to display "binary" type fields.